libraries/AP_ADSB/GDL90_protocol/GDL90_Message_Structs.h

```diff
@@ -0,0 +1,595 @@
+
```

Comment:
Needs attribution - where did this come from?  Is the original author OK with us copying it in here?

Reply:
added